src/ImageSharp/Metadata/Profiles/Exif/Values/ExifNumberArray.cs

```diff
@@ -54,13 +54,25 @@ public override bool TrySetValue(object value)
                 case ushort val:
                     return this.SetSingle(val);
                 case int[] array:
+                {
+                    // workaround for inconsistent covariance of value-typed arrays
+                    if (value.GetType().Equals(typeof(uint[])))
```

Comment:
```suggestion
                    if (value.GetType() == typeof(uint[]))
```
Produces [less code](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIGYACMhgYQYG8aHunGBLAHYYGAFSgBPAMowMANWwAbAK4wAFBGAArGGGEA3RSoCUDLj07UeVhrgDufDGAAWDVQeUwjZ6x28+eYHgwDIL6iiBMAOwM5ADcfv4MgbjBoQDaALoM2FBQ2OIgCYnmRcXWfABmru4qAHQA4jIi4gAOaka1AKIAjkqKuKoYrTAQFapK6RlG06VlJZZzi8TRpPELiz4AvrOLO3PLDHRrGzzb68UAJjAV2EoKWHs+B3Bxs2dW7zx+9CFCohLSOSGGDkdRaHRhDwmPwWRJ2BzOarArznKywubJVJ/dwRA6vVH+TG/DCZbK5fKFAlldEnKyVJEeBpNYaqEwAXjZDCGbVG40mMypGxptKeK2OItOj2KUsSByOMslgqsVxudweSv8z3xiU+3DOmyAA). I believe the JIT is able to play some optimizations in that case.